packages/eas-cli/src/update/configure.ts

```diff
@@ -409,7 +411,7 @@ export async function ensureEASUpdateIsConfiguredAsync({
       workflows,
     });
 
-  if (projectChanged || !hasExpoUpdates) {
+  if (forceNativeConfigSync || projectChanged || !hasExpoUpdates) {
```

Comment:
did this instead of deleting the conditional entirely, because `ensureEASUpdateIsConfiguredAsync` is also called from `eas update`. this minimizes the impact of the change in this PR and ensures that it only applies to `eas update:configure`